docs/usage.md

```diff
@@ -78,6 +78,10 @@ To prevent entire pipeline failures to due a single 'bad sample', nf-core/funcsc
 
 > ⚠️ If a sample does not reach this contig length threshold, you will recieve a warning in your console and `.nextflow.log` file, and no result files will exist for this sample in your results directory.
 
+We also noticed that when annotation is run with Prokka, the `.gbk` file passed to AntiSMASH may produce the error *'translation longer than location allows'* and end the pipeline run.
```

Comment:
```suggestion
Sometimes, when complex samples are run with the `--annotation_tool` set to Prokka, the `.gbk` file passed to AntiSMASH may produce the error *'translation longer than location allows'* and end the pipeline run.
```

Reply:
I don't like using the term 'complex samples', I would rather say large assemblies or very long contigs or something like that

Reply:
The text has been modified already, so that this comment is outdated and no mention of "large, complex, long, samples, assemblies, contigs" is made any more, but the link to the respective discussion on the Prokka repo is given.